src/script/view_model/panel/PanelHeader.tsx

```diff
@@ -34,15 +34,17 @@ const PanelHeader: React.FC<PanelHeaderProps> = ({onGoBack, onClose, title, goBa
   return (
     <div className="panel__header">
       <DragableClickWrapper onClick={onGoBack}>
-        <div className="icon-button" data-uie-name={goBackUie}>
+        <button className="icon-button" data-uie-name={goBackUie} title={goBackUie}>
```

Comment:
I don't think the `title` should be the UIE name. Uie names are never meant to be display to the `user` .

I think we have a translation key for `goBack` somewhere